index.js

```diff
@@ -32,15 +31,14 @@ module.exports = str => {
 		throw new TypeError('Expected a string');
 	}
 
-	// used to see if tabs or spaces are the most used
+	// Used to see if tabs or spaces are the most used
```

Comment:
Why these changes?

Please respect the style of existing code :)

Reply:
It's what XO enforce now.

Reply:
Ok :)